src/features/advanced-content-insights/AdvancedContentInsightsToggle.scss

```diff
@@ -0,0 +1,12 @@
+.AdvancedContentInsightsToggle {
+    display: flex;
+
+    .toggle-container {
+        margin-bottom: 0;
+    }
+}
```

Comment:
if we remove the outer div here:
https://github.com/box/box-ui-elements/blob/c2dfb7efa2db910eee4f3b2cac99b79f73f667f9/src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx#L42-L43

then we could pass in a className prop (`AdvancedContentInsightsToggle`) to `Toggle` and this could simplify to:
```
.AdvancedContentInsightsToggle {
    margin-bottom: 0;
}
```